chart/tests/test_ingress_web.py

```diff
@@ -86,7 +112,7 @@ def test_should_ingress_host_still_work(self):
         assert (
             ["old-host"]
             == jmespath.search("spec.rules[*].host", docs[0])
-            == jmespath.search("spec.tls[0].hosts", docs[0])
+            == list(itertools.chain.from_iterable(jmespath.search("spec.tls[*].hosts", docs[0])))
```

Comment:
can you clarify the need for this change?  if the old behavior should work, shouldn't the old test be good enough?